src/pages/api/comments.ts

```diff
@@ -2,7 +2,7 @@ import { NextApiHandler } from "next";
 
 import { prisma } from "../../shared/prisma-helper";
 import { Comment, NewComment } from "../../types";
-import { getUser } from "./users";
+import { getUser } from "./users/[userId]";
```

Comment:
It’d be nice to avoid extra exports from pages and API routes because they complicate the dependency graph and make it harder to update code with confidence. If all Next.js pages and API routes are leafs in a tree, code maintenance becomes easier. I’d suggest moving `getUser` into `src/???/shared/users` file or folder, where `???` depends on the ‘smallest common parent’ of all dependants.

We can also replace `src/api/users/[userId].ts` with `src/api/users/[user-id].ts` to keep `kebab-case` everywhere.

```ts
const userId = req.query["user-id"];
```

Reply:
1. Moved to services folder.
2. Renamed non-cebab files.